boilermaker/app.py

```diff
@@ -106,13 +115,38 @@ async def publish_task(self, task: Task, delay: int = 0, retries: int = 3):
                 ServiceBusAuthorizationError,
                 ServiceBusAuthenticationError,
             ) as e:
-                encountered_errors.append(e)    
+                encountered_errors.append(e)
         else:
             raise BoilermakerAppException(
                 "Error encountered while publishing task to service bus",
                 encountered_errors,
             )
 
+    async def signal_handler(self, receiver: ServiceBusReceiver, scope: CancelScope):
+        with open_signal_receiver(signal.SIGINT, signal.SIGTERM) as signals:
+            async for signum in signals:
+                if self._current_message is not None:
+                    sequence_number = self._current_message.sequence_number
+                    if signum == signal.SIGINT:
```

Comment:
Can remove this branch since both paths do the same thing and `signal` is `[enum.IntEnum](https://docs.python.org/3/library/enum.html#enum.IntEnum)` so can just pretty print it in the fstring?

Reply:
yeah, I did honestly think about that. I just thought I'd disambiguate here with the log message and also for future contributors. Maybe overoptimizing.

Point taken: I'll collapse. It'll make the simpler.